le.go

```diff
@@ -138,9 +138,22 @@ func (logger *Logger) Flags() int {
 
 // Output does the actual writing to the TCP connection
 func (logger *Logger) Output(calldepth int, s string) error {
-	_, err := logger.Write([]byte(s))
-
-	return err
+	var (
+		err        error
+		waitPeriod = time.Millisecond
+	)
+	for {
+		_, err = logger.Write([]byte(s))
+		if err != nil {
```

Comment:
even the tcp is broken, err still can be nil, so it's not work.

Reply:
If net.SetWriteDeadline is set err would not be nil as the request would timeout. Right ?